### PR TITLE
add OCIO TSC meeting notes

### PR DESCRIPTION
**Pull request by scoopxyz**
Adding first TSC meeting notes on behalf of Kevin Wheatley

### CODE REVIEW

File: docs/tsc/meetings/2019-02-04.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Opening explanation of form and function of TSC by John Mertic.
+* Election of officials/Nominations for roles
+* DW, explained the prior discussion from the Slack #tsc channel
+  * Michael Dolan - Chairperson
+  * Doug Walker for Chief Architect
+* Additional nominations
+  * Sean Cooper - Chairperson
+* Discussion around the concept of Chair + Vice chair and the distinction of only the chair having voting rights on the TAC.
+* LG suggested we may benefit from a singular chair as an efficiency point, rather than the potential of delays by trying to maintain something similar to the current consensus model.
+* John Metric requested candidates provide their background for inclusion in the voting process with the aim of opening voting on the 5th February.
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-02-04.md]: Typo 'Metric' -> 'Mertic'

File: docs/tsc/meetings/2019-02-04.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+Feb. 4, 2019
+
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-02-04.md]: I think that the document should mention the host/organizer:
"Host: John Mertic"

File: docs/tsc/meetings/2019-02-04.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Secretary: Kevin Wheatley
+
+Attendees: Kevin Wheatley, John Mertic, Carl Rand, Larry Gritz, Michael “SPI” Dolan, Sean Cooper, Bernard Lefebvre, Doug Walker, Patrick Hodoul, Thanh Ha
+
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-02-04.md]: [Minor] I think that the list of attendees should be in the alphabetical order (perhaps also adding the Corporation name).

File: docs/tsc/meetings/2019-02-04.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Opening explanation of form and function of TSC by John Mertic.
+* Election of officials/Nominations for roles
+* DW, explained the prior discussion from the Slack #tsc channel
+  * Michael Dolan - Chairperson
+  * Doug Walker for Chief Architect
+* Additional nominations
+  * Sean Cooper - Chairperson
+* Discussion around the concept of Chair + Vice chair and the distinction of only the chair having voting rights on the TAC.
+* LG suggested we may benefit from a singular chair as an efficiency point, rather than the potential of delays by trying to maintain something similar to the current consensus model.
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-02-04.md]: For external readers, I think that we should use complete names.  Refer to line 18 & line 12

**Comment by scoopxyz** [comment]
@hodoulp updated

**Comment by scoopxyz** [comment]
... also this made me realize that I shouldn't edit files from the Imageworks/ASWF repo directly since it creates a branch that no one cares about

**Pull request merged**